UI/ToyCar/ToyCar.Droid/ToyCar.Droid.csproj

```diff
@@ -47,9 +51,12 @@
     <AndroidLinkMode>SdkOnly</AndroidLinkMode>
     <AndroidCreatePackagePerAbi>true</AndroidCreatePackagePerAbi>
     <EmbedAssembliesIntoApk>true</EmbedAssembliesIntoApk>
-    <AotAssemblies>true</AotAssemblies>
-    <EnableLLVM>true</EnableLLVM>
+    <AotAssemblies>false</AotAssemblies>
```

Comment:
was it intentional to remove AOT?

Reply:
No I think it was changed by going to the properties and I didn't notice the changes at first.
I will update the .csproj thanks ;) !

Reply:
It changes back by itself in some editions of VS.

Reply:
Yes that's what I noticed

Reply:
Update done thanks !